docs/application/vstools/install.md

```diff
@@ -1,170 +1,104 @@
-# Installing Visual Studio Tools for Tizen
-
-You need the following components on top of Visual Studio to make Visual Studio Tools for Tizen work:
-
-- VSIX
-
-  Visual Studio extension for Tizen packaging
-
-- Tizen Baseline SDK
-
-  The SDK is shared with the Tizen Studio, and it supports tools (such as Certificate Manager, Device Manager, Emulator, SDB, and on-demand rpm) for developing Tizen .NET applications.
+# Introduction
 
+Visual Studio Tools for Tizen provides Tizen-specific tools to improve your productivity. You can create, build, and debug Tizen applications using Visual Studio. Using Visual Studio Tools for Tizen you can:
+- Create applications for TV, wearable, and mobile profiles
+- Debug your application from Visual Studio
+- Verify the profile performance and memory usage of your application
 
 ## Prerequisites
 
-To work with Visual Studio Tools for Tizen, your computer must have:
-
-- At least 1.5 GB of available disk space
-- Visual Studio 2017
-
-  Visual Studio Tools for Tizen works with all Visual Studio variations, including Community. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
-
-  ![Visual Studio prerequisites](media/prerequisite-vs.png)
-
-- Java Development Kit (JDK) 8
-
-  JDK 8 is needed to use the Tizen Baseline SDK. OpenJDK is not supported.
-
-  Download the JDK 8 from the official [Oracle Web site](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Select the appropriate platform for your hardware architecture and Windows&reg; version. Run the downloaded executable file and follow the displayed instructions.
-  - JDK 9 will be supported soon.
-
-### Emulator Requirements
+To work with Visual Studio Tools for Tizen, you must have Visual Studio 2019.
 
-The Tizen emulator for Visual Studio has the same requirements as the emulator in the Tizen Studio. To check the detailed hardware and software requirements for the Tizen emulator, see [Emulator Requirements](../tizen-studio/setup/prerequisites.md#emulator).
+![Visual Studio Tools for Tizen](media/prerequisite-vs.png)
 
-- Intel&reg; Hardware Acceleration Execution Manager (Intel&reg; HAXM) speeds up the Tizen emulation on Intel-VT-enabled systems. The Intel&reg; HAXM installation is started automatically as part of the Visual Studio Tools for Tizen installation. For more information, see [Hardware Accelerated Execution Manager](../tizen-studio/setup/hardware-accelerated-execution-manager.md).
+> **Note**
+>
+> Visual Studio Tools for Tizen works with all Visual Studio variations, except Express. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
```

Comment:
This note is not correct. The package name is different for VS variations.

Reply:
@journey2w,

Thank you for the comment. Are you referring to the package names for Tizen Tools or .NET development packages?

Because the package name for Tizen Tools is uniform throughout. 

Reference: Package name is same: https://marketplace.visualstudio.com/search?term=tizen&target=VS&category=Tools&vsVersion=&subCategory=All&sortBy=Relevance